modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -104,7 +104,7 @@ To define a DNS configuration for a network interface, you must initially specif
 ====
 You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers unless you manually configured a customized `br-ex` bridge.
 
-For more information, see "Creating a manifest object that includes a customized br-ex bridge" in the _Deploying installer-provisioned clusters on bare metal_ document or the _Installing a user-provisioned cluster on bare metal_ document.
+For more information, see "Creating a manifest object that includes a customized br-ex bridge" in the _Installing on bare metal with the IPI Installer_ document or the _Installing a user-provisioned cluster on bare metal_ document.
```

Comment:
Same comment

Reply:
Original phrasing is, however, inconsistent with all other titles that begin with `Installing...`

Reply:
Addressed